### PR TITLE
Switch to 5:4 aspect ratio for Fronts images

### PR DESCRIPTION
**Pull request by frederickobrien**
## What's changed?

As part of the redesigned homepage project, we want to switch images displayed on fronts to the aspect ratio 5:4 for landscape.  At this phase, we want to enforce 5:4 image crop on replacement images / slideshows to new container types only.

The previous PRs #1590 and #1598 by @dblatcher laid the groundwork for supporting different image crops based on the collection's layout type.  This pull request extends the mechanism to enable 5:4 landscape crop on selected collection type.

This pull request defines the list of collection type (`COLLECTIONS_USING_LANDSCAPE_5_TO_4_TRAILS`) selected for 5:4 image crop.  In future, we can add the new dynamic layout types (and other new container types) to the list.

It makes the following changes for the selected collection type:

1. change `InputImage` component to open Grid with 5:4 landscape crop when users replace the trail image or set slideshow images for the selected collection type.
2. change `ThumbnailSmall` component to use 5:4 image crop for trail image preview even if the card uses the default 5:3 trail image set in the composer.  It is to mimic the auto-crop behaviour on `dotcom-rendering`.
3. change the trail image preview to 5:4 image in the `ArticleMetaForm` when the card is opened for editing.  It requires changes to the `ImageInputImageContainer` and `ImageComponent` components.
4. update the `Card`, `ArticleCard` and `ArticleBody` to pass on the image criteria (whether the collection requires landscape 5:3 or landscape 5:4) to lower level components.

## Implementation notes
As we do not set any layout types for the 5:4 landscape image crop, we do not expect any changes to the existing UI and functions in fronts tool after this PR is merged.

### General
- [ ] 🤖 Relevant tests added
- [X] ✅ CI checks / tests run locally
- [X] 🔍 Checked on CODE

### Client
- [X] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [X] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [X] 📷 Screenshots / GIFs of relevant UI changes included

For the testing purpose, I applied the 5:4 image aspect ratio on `fixed/highlights` container in the tests below.

When we replace the trail image of a card, the fronts tool enforce 5:4 image crop in the Grid:

The trail image preview in the collection is also set to 5:4 crop even if the trail image (which is 5:3) is not replaced.  The trail image in clipboard remains 5:3 ratio.

Slideshows have also been changed to enforce 5:4 image crop and their preview images show 5:4 aspect ratio:

### CODE REVIEW

File: fronts-client/src/components/inputs/InputImage.tsx
Comment (paperboyo) [reply fronts-client/src/components/inputs/InputImage.tsx:558]: Wot Charlotte said.

And to restrict to only this new one, we need both arguments, so `&cropType=new&customRatio=new,5,4`.
Now, a kingdom for a good name…

File: fronts-client/src/components/image/ImageInputImageContainer.tsx
Comment (lindseydew) [inline fronts-client/src/components/image/ImageInputImageContainer.tsx:53]: This may also be a good candidate for a unit test

File: fronts-client/src/components/inputs/InputImage.tsx
Comment (lindseydew) [inline fronts-client/src/components/inputs/InputImage.tsx:578]: This could also be a candidate for a unit test

**Review by lindseydew — COMMENTED** [review]
Nice, this LGTM. Just one non-blocking suggestion of unit tests in a couple of places, if the existing infrastructure supports being able to add those easily.

 I'll hold off approving as I'm not a code owner of this repo

File: fronts-client/src/components/inputs/InputImage.tsx
Comment (paperboyo) [reply fronts-client/src/components/inputs/InputImage.tsx:581]: Super-nit! Naming is hard. But this looks weird:

image

Maybe just `&cropType=Landscape&customRatio=Landscape,5,4`

image

Or `Front` or whatevs?

Comment (waisingyiu) [reply fronts-client/src/components/inputs/InputImage.tsx]: Just `&cropType=Landscape&customRatio=Landscape,5,4` looks good to me!

**Review by Georges-GNM — APPROVED** [review]
This looks good to me - notably, testing on code, it doesn't look like there's been any unexpected changes to other containers, so feels fine to merge.

**Pull request merged**
